### PR TITLE
http://tgs.mindfine.com/ConcernedFroge/cry_10078c4d.gif

### PR DESCRIPTION
**Pull request by Phyco-Ninja**
Again Gban 🤔 Added Admin and non admin list to avoid flood wait for checking admin rights of every chat. 😐😑

### CODE REVIEW

File: userge/plugins/admin/gban.py [hunk 1]
```diff
@@ -309,27 +321,27 @@ async def gban_at_entry(message: Message):
...
+                                f"**ID:** `{user_id}`\n**Reason:** `{reason}`\n\n"
+                                "**Quick Action:** Banned"),
+                            CHANNEL.log(
+                                r"\\**#Antispam_Log**//"
+                                "\n\n**GBanned User $SPOTTED**\n"
+                                "**$SENRTY #CAS BAN**"
+                                f"\n**User:** [{first_name}](tg://user?id={user_id})\n"
+                                f"**ID:** `{user_id}`\n**Reason:** `{reason}`\n**Quick Action:**"
+                                f" Banned in {message.chat.title}\n\n$AUTOBAN #id{user_id}")
+                        )
```
Comment (rking32) [inline userge/plugins/admin/gban.py:344]: why add indent like this ?

at first check user is gbanned
```python
            gbanned = await GBAN_USER_BASE.find_one({'user_id': user_id})
            if gbanned:
```
at second check spamwatch
```python
            elif Config.ANTISPAM_SENTRY and Config.SPAM_WATCH_API:
```
at finally it should check this
```python
            else:
                res = requests.get(f'https://api.cas.chat/check?user_id={user_id}').json()
```
but now it breaked :thinking:

Comment (Phyco-Ninja) [reply userge/plugins/admin/gban.py:344]: CAS should come in API Bans block 🤔🤔

Comment (rking32) [reply userge/plugins/admin/gban.py:344]: ohh

**Comment by rking32** [comment]
now looks good ?

**Comment by Phyco-Ninja** [comment]
Yea

**Pull request merged**